patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1359,12 +1359,21 @@
 +        return false;
 +    }
 +
++    /**
++     * Set the PathNodeType for this block. null for vanilla behavior.
```

Comment:
Suggestion for the comment:
`Get the PathNodeType for this block. Return null for vanilla behavior.`